src/osx/cocoa/dnd.mm

```diff
@@ -244,6 +244,10 @@ virtual CFDataRef DoGetData(wxDataFormat::NativeFormat type) const
 
 #if wxUSE_DRAG_AND_DROP
 
+// the new API will be used with 1, with 0 the old API is in place
+
+#define wxOSX_USE_DRAG_SESSION 1
+
```

Comment:
I know it's done in several other places in wxOSX code, but do we really need this? I'd just switch to the new code, with Git (or any other VCS, to be fair) it's trivial to revert this commit later if needed, so having both versions of the code doesn't really gain us much, but has the obvious drawbacks.

I'd just replace the old code with the new one unless we have some really good reasons to believe we're going to need the old version in the future.

Reply:
> I'd just replace the old code with the new one unless we have some really good reasons to believe we're going to need the old version in the future.

yes, before the final commit I'd remove the old version, I just use this for testing behaviour, switching on and off